src/components/EstimateQuarter.tsx

```diff
@@ -0,0 +1,114 @@
+import { Button, useClickOutside } from '@taskany/bricks';
+import { gray6, gapXs } from '@taskany/colors';
+import { useState, useRef, useCallback, Dispatch, SetStateAction, useEffect } from 'react';
+import styled from 'styled-components';
+
+import { quarters, endOfQuarter, createLocaleDate, yearFromDate } from '../utils/dateTime';
+import { Estimate, Option, QuartersKeys } from '../types/estimate';
+import { useLocale } from '../hooks/useLocale';
+
+import { EstimateOption } from './EstimateOption';
+
+const StyledQuarters = styled.div`
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    gap: ${gapXs};
+`;
+
+const StyledCheckableButton = styled(Button)<{ checked?: boolean }>`
+    background-color: ${({ checked }) => checked && gray6};
```

Comment:
We don't have this style in bricks? Create issue if not and put comment with link here

Reply:
https://github.com/taskany-inc/bricks/pull/418